Documentation/operator.md

```diff
@@ -49,6 +49,8 @@ Usage of ./operator:
     	Value used by the operator to filter Alertmanager, Prometheus, PrometheusAgent and ThanosRuler objects that it should reconcile. If the value isn't empty, the operator only reconciles objects with an operator.prometheus.io/controller-id annotation of the same value. Otherwise the operator reconciles all objects without the annotation or with an empty annotation value.
   -deny-namespaces value
     	Namespaces not to scope the interaction of the Prometheus Operator (deny list). This is mutually exclusive with --namespaces.
+  -disable-unmanaged-prometheus-configuration additionalScrapeConfigs
```

Comment:
hmm, not sure why we have `additionalScrapeConfigs` here but it looks like removing the backticks in the help text should fix it :exploding_head: